Content.Server/_DV/CosmicCult/Abilities/Colossus/CosmicEffigySystem.cs

```diff
@@ -0,0 +1,102 @@
+using System.Numerics;
+using Content.Server._DV.CosmicCult.Components;
+using Content.Server.Actions;
+using Content.Server.Objectives.Components;
+using Content.Server.Objectives.Systems;
+using Content.Server.Popups;
+using Content.Shared._DV.CosmicCult;
+using Content.Shared._DV.CosmicCult.Components;
+using Content.Shared._DV.CosmicCult.Components.Examine;
+using Content.Shared.Damage;
+using Content.Shared.Maps;
+using Content.Shared.Mind;
+using Content.Shared.Stunnable;
+using Content.Shared.Warps;
+using Robust.Shared.Map;
+using Robust.Shared.Map.Components;
```

Comment:
Using directive is unnecessary.
```suggestion
using System.Numerics;
using Content.Server.Actions;
using Content.Server.Objectives.Components;
using Content.Server.Objectives.Systems;
using Content.Server.Popups;
using Content.Shared._DV.CosmicCult;
using Content.Shared._DV.CosmicCult.Components;
using Content.Shared.Damage;
using Content.Shared.Maps;
using Content.Shared.Mind;
using Content.Shared.Stunnable;
using Content.Shared.Warps;
using Robust.Shared.Map;
using Robust.Shared.Map.Components;
```